ps_checkout.php

```diff
@@ -44,12 +44,12 @@ class Ps_checkout extends PaymentModule
     // hook list used by the module
     const HOOK_LIST = [
         'paymentOptions',
-        'paymentReturn',
+        'displayPaymentReturn',
         'actionFrontControllerSetMedia',
         'actionOrderSlipAdd',
-        'orderConfirmation',
+        'displayOrderConfirmation',
```

Comment:
does this will still work even tho the method is still called `hookOrderConfirmation` ?

Reply:
Yes, real name and alias are tested on
* Hook::getIdByName()
* Hook::isHookCallableOn()
* Hook::callHookOn()

Moreover on Hook::registerHook() real hook id is used even with alias thanks to Hook::getIdByName()

Better to change hookorderConfirmation to hookDisplayOrderConfirmation indeed